docs/theming/dark-mode.md

```diff
@@ -14,76 +17,166 @@ inlineHtmlPreviews: true
+
+### Always
+
+The default theme provided with Ionic Framework is a light theme, consisting of a light background and dark text. However, the default theme can be changed to the dark theme by importing the following stylesheet in the appropriate files:
 
-The modern way to enable dark mode is by using the [CSS media query for the user's preferred color scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme). This media query will hook into the system setting of the user's device and apply the theme if dark mode is enabled. Ionic Framework starters use this method to include the dark theme.
+<Tabs groupId="framework" defaultValue="angular" values={[{ value: 'angular', label: 'Angular' }, { value: 'javascript', label: 'Javascript' }, { value: 'react', label: 'React' }, { value: 'vue', label: 'Vue' }]}>
+
+<TabItem value="angular">
 
 ```css
-@media (prefers-color-scheme: dark) {
-  :root {
-    /* Dark mode variables go here */
-  }
-}
+@import '~@ionic/angular/css/themes/dark.always.css';
 ```
 
-The `prefers-color-scheme` media query is supported by [all modern browsers](https://caniuse.com/#feat=prefers-color-scheme). Users will not be able to benefit from having the dark theme applied using this media query in certain browsers, however, the dark theme can still be applied by using a [CSS class fallback](#css-class-fallback) if support for older browsers is needed.
+</TabItem>
+<TabItem value="javascript">
 
-## CSS Class Fallback
+```ts
+import '@ionic/core/css/themes/dark.always.css';
+```
 
-As a fallback method for devices that don't support the media query, the dark mode can be applied by styling a CSS selector and applying the class to the document body.
+</TabItem>
+<TabItem value="react">
 
-```css
-@media (prefers-color-scheme: dark) {
-  :root {
-    /* Dark mode variables go here */
-  }
-}
+```tsx
+import '@ionic/react/css/themes/dark.always.css';
+```
 
-/* Fallback for older browsers or manual mode */
-body.dark {
-  /* Dark mode variables go here */
-}
+</TabItem>
+<TabItem value="vue">
+
+```ts
+import '@ionic/vue/css/themes/dark.always.css';
 ```
 
-With the variables targeting the `body.dark` selector, all that is needed now is to add the class to the `<body>` in the app. This can be done in a variety of ways depending on the framework your app is built with.
+</TabItem>
+
+</Tabs>
+
+This sets the [application colors](/docs/theming/themes#application-colors) and [stepped colors](/docs/theming/themes#stepped-colors) on the [`:root`](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) selector.
+
+:::caution Important
+Avoid targeting the `.ios` or `.md` selectors to override the Ionic dark theme, as these classes are added to each component and will take priority over globally defined variables. Instead, we can target the mode-specific classes on the `:root` element.
+:::
+
+:::info Demo
+This demo will always display the dark theme, regardless of the system settings for dark mode.
+:::
```

Comment:
I can remove the info, I mostly just kept it there to bring attention to how to change the system settings to view the dark mode, and then also added it to the "always" demo to be consistent:

![Screenshot 2024-01-18 at 6 22 41 PM](https://github.com/ionic-team/ionic-docs/assets/6577830/79b21ded-4f2a-4ae7-814f-5052f93e2c39)

This part:

> Not sure how to change the system settings? Here's how to enable dark mode on [Windows 11](https://support.microsoft.com/en-us/windows/change-colors-in-windows-d26ef4d6-819a-581c-1581-493cfcc005fe) and on [macOS](https://support.apple.com/en-us/HT208976).

I feel like it's easy to miss otherwise and people might not immediately know how to toggle the dark theme on. 

Here is what v7 looked like:

![Screenshot 2024-01-18 at 6 24 43 PM](https://github.com/ionic-team/ionic-docs/assets/6577830/e444064c-611c-408e-98d3-cfceb97a827c)

Reply:
Removed here: https://github.com/ionic-team/ionic-docs/pull/3384/commits/a00ba1120602efb26c2eba229611218b0c4f210c

Reply:
Hmm, then let's only keep the "Not sure how to..." part in the info block. I also feel that this can be easily missed.

Reply:
Done https://github.com/ionic-team/ionic-docs/pull/3384/commits/65fd8cb8fca306e0eec7c96b57bd6c2f9b60a0cf